noisyatom/catalog/models.py

```diff
@@ -24,10 +25,13 @@ class Meta:
 
     def __unicode__(self):
         return self.name
-
+'''
     @models.permalink
     def get_absolute_url(self):
         return 'catalog_category', (), {'category_slug': self.slug}
+'''
```

Comment:
Why did you comment out this code?
If it's not needed, should we just remove it? Were you planning on putting it back after some testing?

Reply:
Thanks! I did remove the old code. It won't work anymore with Django 2.2.